### ISSUE TITLE
Document browser (in-)compatibility

### ISSUE THREAD

**Issue by bensheldon** [ianks/yagni_json_encoder#2]
I apologize if I overlooked it, but could you please define "older browsers" in the Readme? I tried googling for what browsers necessitate the extra encoding, but I wasn't very successful.

**Comment by ianks** [comment]
Yes you are right. I do not have access to these older browsers, so testing will be tough. The `\u2028` and `\u2029` represent line seperators in unicode. In my limited testing, current versions of Chromium (`Version 48.0.2564.109 (64-bit)`) and Firefox(`Mozilla Firefox 44.0.2`) parse this correctly: 

`JSON.parse('{"test": "\u2029", "test2": "\u2028"}')`

If some people could document other browsers here, that would be wonderful!
